src/hintingtt.js

```diff
@@ -0,0 +1,3043 @@
+/* A TrueType font hinting interpreter.
+*
+* (c) 2017 Axel Kittenberger
+*
+* This file is covered by the MIT (expat) License,
+* see the LICENSE file in the root of this project.
```

Comment:
I don't if it's relevant to add the mention to the MIT license as the LICENSE file is here for that